diff --git a/fuse/fuse_int.c b/fuse/fuse_int.c
index b8736e4..4273107 100644
--- a/fuse/fuse_int.c
+++ b/fuse/fuse_int.c
@@ -189,7 +189,7 @@ static int fuse_open(const char *path, struct fuse_file_info *fi)
 	struct afp_volume * volume=
 		(struct afp_volume *)
 		((struct fuse_context *)(fuse_get_context()))->private_data;
-	unsigned char flags = AFP_OPENFORK_ALLOWREAD;
+	unsigned char flags = O_RDONLY;
 
 	log_fuse_event(AFPFSD,LOG_DEBUG,
 		"*** Opening path %s\n",path);
